toolchain/sem_ir/file.h

```diff
@@ -70,6 +70,30 @@ class File : public Printable<File> {
     return types().GetAs<PointerType>(pointer_id).pointee_id;
   }
 
+  struct IntTypeInfo {
+    bool is_signed;
+    IntId bit_width = IntId::Invalid;
+  };
+
+  // Compute the core integer type information from a type ID.
+  //
+  // TODO: When we don't have a builtin int type mixed with actual `IntType`
+  // instructions, clients should directly query the `IntType` instruction to
+  // compute this information.
+  auto GetIntTypeInfo(TypeId int_type_id) const -> IntTypeInfo {
+    auto inst_id = types().GetInstId(int_type_id);
+    if (inst_id == InstId::BuiltinIntType) {
+      return {true, ints().LookupSigned(llvm::APInt(/*numBits=*/64, 32))};
```

Comment:
```suggestion
      return {.is_signed = true, .bit_width = ints().LookupSigned(llvm::APInt(/*numBits=*/64, 32))};
```

I think we'd agreed to use designated initializers for struct types (https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md#syntax-and-formatting)